ts/features/bonus/cdc/screens/CdcBonusRequestSelectResidence.tsx

```diff
@@ -62,11 +61,9 @@ const CdcBonusRequestSelectResidence = () => {
     >
       <SafeAreaView style={IOStyles.flex}>
         <ScrollView style={IOStyles.horizontalContentPadding}>
-          <H1>{I18n.t("bonus.cdc.bonusRequest.selectResidence.header")}</H1>
+          <H2>{I18n.t("bonus.cdc.bonusRequest.selectResidence.header")}</H2>
           <VSpacer size={16} />
-          <H4 weight={"Regular"}>
-            {I18n.t("bonus.cdc.bonusRequest.selectResidence.info")}
-          </H4>
+          <H6>{I18n.t("bonus.cdc.bonusRequest.selectResidence.info")}</H6>
```

Comment:
In that case, we have `H3` and `H4` that became H6, which appears to be an error but we can ignore it for now as `CdC` has been a frozen project for a long time